core/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinderFactory.java

```diff
@@ -302,99 +304,77 @@ private <T> Binder<T, ConsumerProperties, ProducerProperties> getBinderInstance(
 			this.binderInstanceCache.put(configurationName,
 					new SimpleImmutableEntry<>(binder, binderProducingContext));
 		}
-		logger.info("Retrieving cached binder: " + configurationName);
+		logger.trace("Retrieving cached binder: " + configurationName);
 		return (Binder<T, ConsumerProperties, ProducerProperties>) this.binderInstanceCache
 				.get(configurationName).getKey();
 	}
 
-	@SuppressWarnings("unused")
```

Comment:
**[UNRELATED]** This has been in here for quite a while, its unused, the file has grown very large, we can pull it back out of git history if we ever really need it.